app/src/main/res/values-zh-rCN/strings.xml

```diff
@@ -0,0 +1,54 @@
+<resources>
+    <string name="app_name">Scroball</string>
+
+    <!-- Strings related to Settings -->
+    <string name="saved_session_key">会话密钥</string>
```

Comment:
This needs to be left as sessionKey as well (it's for storage, not localisation).